egs2/babel/asr1/README.md

```diff
@@ -0,0 +1,86 @@
+# BABEL RECIPE
+
+## Setting Up Experiments
```

Comment:
We should remove or modify this file.
Maybe we can remove them except this part:

```
### Examples
`./run.sh --langs "102" --recog "102"`
  This will create a network trained on 102 (assamese) and will test it on 102.

`./run.sh --langs "102 103" --recog " 102 103"`
  This will create a network trained on 102 and 103 (bengali) and tests on both.
```